Jenkinsfile

```diff
@@ -34,6 +34,7 @@ pipeline {
               sh("composer config --global github-oauth.github.com ${composertoken}")
             }
             sh '''
+              git config --global --add safe.directory '*'
```

Comment:
Why are we including a safe directory? Are you experiencing pipeline errors?

Reply:
@avikarshasaha yes, there was a pipeline issue. See https://core.cloudbees.ais.acquia.io/mautic-jenkins/job/mc-cs-plugins/job/mc-cs-plugin-slooce/job/PR-22/1/console

Reply:
Yes, I was facing[ this](https://acquia.slack.com/archives/CJHAV5Y91/p1675068440463909) exact error. I followed the solution suggested by @himanshu8acquia